org.eclipse.draw2d/src/org/eclipse/draw2d/ConnectionEndpointLocator.java

```diff
@@ -1,5 +1,5 @@
 /*******************************************************************************
- * Copyright (c) 2000, 2010 IBM Corporation and others.
+ * Copyright (c) 2000, 2023 IBM Corporation and others.
```

Comment:
Just a reminder, we're in 2024 already 😉

Reply:
It seems I'm still living in the past ;-) will fix it.